Nodejs/Product/Npm/SPI/NpmCommander.cs

```diff
@@ -211,23 +199,15 @@ private class NmpCommandRunner : IDisposable {
             return await UpdatePackagesAsync(new List<IPackage>());
         }
 
-        private async Task<bool> UpdatePackagesAsync(IEnumerable<IPackage> packages, bool global) {
-            return await DoCommandExecute(true,
+        public Task<bool> UpdatePackagesAsync(IEnumerable<IPackage> packages) {
+            return DoCommandExecute(true,
```

Comment:
In my understanding, because we already have a task, a function with a single line return statement does not need to await because that essentially just rewraps the same task. I can revert these change if we want to be more cautious though.